main.cpp

```diff
@@ -185,10 +329,38 @@ struct CarWash //                                   1) define an empty struct fo
     3) change volume
  */
 
+
+struct Oscillator       
+{
+    //has volume
+    int volume = 127; //3) member variables with relevant data types.
+    //has sin
+    int sin = 127; //3) member variables with relevant data types.
+    //has square
+    int square = 23; //3) member variables with relevant data types.
+    //has triangle
+    int triangle = 27; //3) member variables with relevant data types.
+    //has noise
+    int noise = 127; //3) member variables with relevant data types.
```

Comment:
What kind of information are these properties supposed to represent?
- [ ] revise.  pick properties that provide information and name them accordingly.

Reply:
These are the properties of my oscillator.